pkg/stub/handler.go

```diff
@@ -109,3 +118,50 @@ func (h *Handler) syncIngressUpdate(ci *ingressv1alpha1.ClusterIngress) error {
 
 	return nil
 }
+
+func (h *Handler) deleteIngress(ci *ingressv1alpha1.ClusterIngress) error {
```

Comment:
Error returned by deleteIngress is ignored by the caller. We are not going to propagate the error to the user (which is the case here), just log the error instead of returning the error or at least the caller need to log the error.